### PR TITLE
Median in a running stream of integers added

### PR DESCRIPTION
**Pull request by gargVader**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

C++ program for finding median in a running stream of integers added. New section for heaps added (confirmed on slack).

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1631

### CODE REVIEW

**Comment by gargVader** [comment]
Please merge #1601 before this. I think I mistakenly updated my master branch too while making changes in other branch. That's why its showing commits from that branch here. I am really sorry for this mishap

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -124,6 +124,10 @@
 - [Check for balanced parenthesis](stack/Check_for_balanced_parenthesis.cpp)
 - [Stock Span Problem](stack/Stock_span_problem.cpp)
 
+## Heaps based problems
```
Comment (HarshCasper) [inline C-Plus-Plus/README.md]: Just mention Heaps

File: C-Plus-Plus/heaps/Median_in_a_running_stream_of_integers.cpp [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+Time Complexity: O(nlogn)
+Space Complexity: O(n)
+*/
+
+#include <iostream>
+#include<queue>
+using namespace std;
+
+int main() {
```
Comment (HarshCasper) [inline C-Plus-Plus/heaps/Median_in_a_running_stream_of_integers.cpp]: Shift your entire logic to a different function. And use main() just for the Driver Code.

File: C-Plus-Plus/heaps/Median_in_a_running_stream_of_integers.cpp [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+	priority_queue<int> maxHeap;
+	priority_queue<int, vector<int>, greater<int>> minHeap;
+
+	// Input first number
+	cin >> n;
+	// Push in maxheap
+	maxHeap.push(n);
+
+	// current median
+	double curr = n; // When count of numbers is 1, median is equal to the number itself
```
Comment (HarshCasper) [inline C-Plus-Plus/heaps/Median_in_a_running_stream_of_integers.cpp]: Use comments above the LOC. Not beside it.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**